contracts/smart-account/interfaces/modules/ISecurityPolicyManagerPlugin.sol

```diff
@@ -0,0 +1,81 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.20;
+
+import {ISecurityPolicyPlugin} from "./ISecurityPolicyPlugin.sol";
+import {ISecurityPolicyManagerPluginEventsErrors} from "./ISecurityPolicyManagerPluginEventsErrors.sol";
+
+address constant SENTINEL_MODULE_ADDRESS = address(0x1);
+
+/**
+ * @title Security Policy Manager Plugin
+ * @author @ankurdubey521
+ * @notice Execution Phase Plugin responsible for enforcing security policies during plugin installation on the smart account
+ */
+interface ISecurityPolicyManagerPlugin is
+    ISecurityPolicyManagerPluginEventsErrors
+{
+    /**
+     * @dev Enables the security policy for the smart account. Used during the setup process.
+     * @param _policy The security policy to be enabled
+     */
+    function enableSecurityPolicy(ISecurityPolicyPlugin _policy) external;
+
+    /**
+     * @dev Enables the security policies for the smart account. Used during the setup process.
+     * @param _policies The security policies to be enabled
+     */
+    function enableSecurityPolicies(
+        ISecurityPolicyPlugin[] calldata _policies
+    ) external;
+
+    /**
+     * @dev Disables the security policy for the smart account.
+     * @param _policy The security policy to be disabled.
+     * @param _pointer The address of the security policy preceeding _policy in the list of enabled modules.
+     */
+    function disableSecurityPolicy(
+        ISecurityPolicyPlugin _policy,
+        ISecurityPolicyPlugin _pointer
+    ) external;
+
+    /**
+     * @dev Disables the security policies for the smart account.
+     * @param _start The first iterm in the list to be disabled.
+     * @param _end The last iterm in the list to be disabled.
```

Comment:
typo here as well. 

_start, _end and _pointer will all be addresses right?

Reply:
yup, they will be pointers